### ISSUE TITLE
Install script doesn't install and configure tomcat on Ubuntu 20

### ISSUE THREAD

**Issue by woodkri** [cassproject/CASS#198]
It checks for 16 and 18.

**Comment by abhinavdayal** [comment]
Yes, Today I modified the script to install on Ubuntu 22.04. You can do the same for 20.

However after installing the 1.4 version, and setting us https, the web sockets were not working. Earlier I deployed the 1.5 version, there wss worked. However the pom.xml is missing in 1.5, so maven could not build.

Note that in the script I have commented some parts
1. I am not overwriting the script on md5sum mismatch
2. I am not removing the CASS repo after installation, just in case I have to make custom changes.

[cassInstall.txt](https://github.com/cassproject/CASS/files/10186788/cassInstall.txt)
